producerapp/src/main/java/de/tuberlin/dos/monitoring/producer/Producer.java

```diff
@@ -57,8 +118,9 @@ private static void runMessageLoop(
 	private static KafkaProducer<String, String> createProducer() {
 
 		Properties properties = new Properties();
-		properties.setProperty(ProducerConfig.BATCH_SIZE_CONFIG, Integer.toString( 1024));
-		properties.setProperty(ProducerConfig.LINGER_MS_CONFIG, "5");
+		// batch size 64KB and +20ms higher linger for higher troughput
+		properties.setProperty(ProducerConfig.BATCH_SIZE_CONFIG, Integer.toString(64 * 1024));
+		properties.setProperty(ProducerConfig.LINGER_MS_CONFIG, "20");
```

Comment:
Bitte so beibehalten, ist wichtig, damit Nachrichten möglichst gleichmäßig auf Partitionen verteilt werden.

```suggestion
properties.setProperty(ProducerConfig.BATCH_SIZE_CONFIG, Integer.toString( 1024));
properties.setProperty(ProducerConfig.LINGER_MS_CONFIG, "5");
```